src/components/[guild]/JoinModal/JoinModal.tsx

```diff
@@ -53,6 +53,10 @@ const JoinModal = ({ isOpen, onClose }: Props): JSX.Element => {
       const ConnectComponent = customJoinStep[platform]
       return <ConnectComponent key={platform} />
     }
+
+    // TODO: Temporary solution, we'll handle this automatically after the rewards/platforms refactor
```

Comment:
This should be done on BE at the requiredPlatforms filter, as POAP is filtered there too